tools/update_runnablejobs.py

```diff
@@ -62,19 +63,25 @@ def update_runnableapi():
         else:
             print "It's going to update your runnable jobs data."
             data = query_the_runnablejobs(new_timestamp, task_id)
-            new_jobs = add_jobs_to_jobpriority(new_data=data,
-                                               priority=1,
-                                               timeout=0,
-                                               set_expired=True)
+            # We're calling with set_expired to True since we expect the new
+            # runabble file might have new jobs
+            # XXX: How do we know that in this case we're going to have new jobs instead of
+            #      the other case below? Does the timestamp in meta that runnable apis give us
+            #      mean that it actually has new data?
+            #add_jobs_to_jobpriority(new_data=data,
+            #                        priority=1,
+            #                        timeout=0,
+            #                        set_expired=True)
             with open(path, 'w') as f:
-                json.dump(data, f)
+                json.dump(data, f, indent=2, sort_keys=True)
     else:
         print "It's going to help you download the runnable jobs file."
         data = query_the_runnablejobs(new_timestamp, task_id)
         if data:
             with open(path, 'w') as f:
-                json.dump(data, f)
-            add_jobs_to_jobpriority(new_data=data, priority=5, timeout=5400)
+                json.dump(data, f, indent=2, sort_keys=True)
+            # We're calling this without setting set_expired to True
+            #add_jobs_to_jobpriority(new_data=data, priority=5, timeout=5400)
```

Comment:
this is by design- we do not have a pre-existing file and we assume we are bootstrapping the system.